### PR TITLE
[security] Fix, let admin's be able to reset user passwords on AUTH_DB

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

- [X] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Let Admin user's be able to reset password when on AUTH_DB

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
image

After:
image

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/security/manager.py [hunk 1]
```diff
@@ -81,7 +81,11 @@ def __init__(self, **kwargs):
 PermissionModelView.list_widget = SupersetSecurityListWidget
 
 # Limiting routes on FAB model views
-UserModelView.include_route_methods = RouteMethod.CRUD_SET | {"userinfo"}
+UserModelView.include_route_methods = RouteMethod.CRUD_SET | {
+    "userinfo",
+    "action_post",
+    "action",
+}
```
Comment (villebro) [inline superset/security/manager.py]: Should we be using `constants.RouteMethod.ACTION_POST` etc here?

Comment (dpgaspar) [reply superset/security/manager.py]: oh! yes, right you are

File: superset/security/manager.py [hunk 2]
```diff
@@ -81,7 +81,11 @@ def __init__(self, **kwargs):
 PermissionModelView.list_widget = SupersetSecurityListWidget
 
 # Limiting routes on FAB model views
-UserModelView.include_route_methods = RouteMethod.CRUD_SET | {"userinfo"}
+UserModelView.include_route_methods = RouteMethod.CRUD_SET | {
+    RouteMethod.ACTION,
+    RouteMethod.ACTION_POST,
+    "userinfo",
```
Comment (villebro) [inline superset/security/manager.py]: The `userinfo` looks lonely here with the constants. Wondering if we should collect these somewhere, too.

Comment (dpgaspar) [reply superset/security/manager.py]: We can of course, did a quick search and found a bunch "lonely" routes. Like `download_dashboards`, `bulk_delete`, `table_metadata`. Would say that the idea here is that non repeatable ones, get referenced in place.

Comment (villebro) [reply superset/security/manager.py]: Perhaps not something that needs to be addressed now (not really sure which is better). LGTM

**Review by villebro — APPROVED** [review]

**Review by craig-rueda — APPROVED** [review]

**Pull request merged**